tests/test_workspace_semantic_release_decision.py

```
@@ -0,0 +1,2229 @@
+
+    assert tree_snapshot(harness.state_root) == before
+    assert not _binding_path(harness.state_root).parent.parent.exists()
+
+
+@pytest.mark.parametrize("missing_kind", ["parent", "file"])
+def test_optional_stable_missing_read_checks_deadline_after_observation(
+    tmp_path: Path,
+    monkeypatch: pytest.MonkeyPatch,
+    missing_kind: str,
+) -> None:
+    harness, _generations, store, _gc = _runtime(tmp_path)
```

Ruff (0.16.1)

[warning] Unpacked variable `harness` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)